docs/users/extend/config_yaml.md

```diff
@@ -20,9 +20,9 @@ the .ddev/config.yaml is the primary configuration for the project.
-| composer_root                    | Relative path to the composer root directory from the project root                         | This is the directory which contains the composer.json and where all Composer related commands are executed.                                                                                                                                                                                                                                                                                                                                   |
-| composer_version                 | version of composer to use in web container and `ddev composer`                            | It defaults to composer v2; you can set it to "" or "2" (default) for composer v2 or "1" for composer v1 to use the latest major.minor.patch versions available at the time your currently installed ddev version was bundled and released. Note that the bundled default version might be behind the latest available composer release. Alternatively, an explicit composer version may be specified, for example `composer_version: 1.0.22`. |
-| nodejs_version                   | version of nodejs to use in web container as "system" version                              | It defaults to current LTS version; you can currently set it to "12", "14", "16", "17", "18". Note that nvm is also available inside the container and via `ddev nvm` and can be set to any valid version including very old versions.                                                                                                                                                                                                          |
+| composer_root                    | Relative path to the Composer root directory from the project root                         | This is the directory which contains the composer.json and where all Composer related commands are executed.                                                                                                                                                                                                                                                                                                                                   |
+| composer_version                 | version of Composer to use in web container and `ddev composer`                            | It defaults to Composer v2; you can set it to "" or "2" (default) for Composer v2 or "1" for Composer v1 to use the latest major version available at the time your container is built. It is also possible to select a minor version for example `2.2` which will install the latest release of that branch. Alternatively, an explicit Composer version may be specified, for example `1.0.22`. Finally, it is also possible to use one of the key words `stable`, `preview` or `snapshot` see Composer documentation. |
```

Comment:
This stuff should also be added to template.go, where the comments for config.yaml are added.

Reply:
It also needs to be added (at least a little) to the `--composer-version` help.

Reply:
Yeah, didn't think about these two, will add it...

Reply:
Templates is changed now too.